promgen/prometheus.py

```diff
@@ -132,12 +132,21 @@ def render_config(service=None, project=None):
         if exporter.path:
             labels["__metrics_path__"] = exporter.path
 
+        for label in exporter.exporterlabel_set.all():
+            if label.name not in labels:
+                label_name = '__param_{}'.format(label.name) if label.is_parameter else label.name
```

Comment:
As mentioned above, I think we don't want to use `is_parameter` but filter on `__param_` instead.